### ISSUE TITLE
libdrm.so.2: cannot open shared object file

### ISSUE THREAD

**Issue by pavo976** [xorilog/docker-telegram#12]
https://hub.docker.com/r/xorilog/telegram

```
docker images|grep xorilog|grep latest
xorilog/telegram       latest              c3b746caa185        28 hours ago        360MB
```
```
#!/usr/bin/env bash

docker run --rm -it --name telegram \
       --hostname=$(hostname) \
       --device /dev/snd \
       -e DISPLAY=unix$DISPLAY \
       -v /tmp/.X11-unix:/tmp/.X11-unix \
       -v "/home/$(whoami)/.Xauthority:/home/user/.Xauthority" \
       -v /etc/localtime:/etc/localtime:ro \
       -v /opt/teleg/data/:/home/user/.local/share/TelegramDesktop/ \
       xorilog/telegram
```

```
./docker-teleg.sh
/usr/bin/Telegram: error while loading shared libraries: libdrm.so.2: cannot open shared object file: No such file or directory
```

**Comment by xorilog** [comment]
Hello @pavo976, does installing libdrm2 in the container fixes your issue ?

**Comment by xorilog** [comment]
Fixed by #13
